recipes/reviewer/build.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+
+mkdir -p "${PREFIX}/bin"
+wget "https://github.com/Illumina/REViewer/releases/download/v${PKG_VERSION}/REViewer-v${PKG_VERSION}-linux_x86_64.gz
```

Comment:
The file must be already downloaded (and uncompressed) by the recipe.

Reply:
I more or less did copy-paste of the `delly` recipe. I'll go ahead and remove this part.

Reply:
You could temporarily use `ls -laR` to see what's available for the build.